### ISSUE TITLE
[Visual requirements - ADF - Azure data factory] List items have insufficient contrast ratio on Azure data factory window.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/AzureStorageExplorer#3842]
**User Experience:**
Visual impaired user will find difficulty to see the control or element if that control or element have insufficient color contrast as user won't be able to perform functionality over such control.

**Test Environment:**
Application Version: 1.16.0-dev
Build Number: 20200921.1

**Pre-Requisite:**
Navigate to the storage expoler.exe and select it  

**Repro Steps:​**
- Navigate and select Add data factory control at left panel.
- Now, navigate to list item of various control like Account, Subscription, etc. on Azure data factory window.
- Verify contrast ratio of list items of various control like Account, Subscription, etc. on Azure data factory window.

**Actual Result:**
List items have insufficient contrast ratio of 3.2:1 for various control like Account, Subscription, etc. on Azure data factory window and due to this, visual impaired user will find difficulty to see such list items.

**Expected Result:**
List items must have sufficient contrast ratio of 4.5:1 for various control like Account, Subscription, etc. on Azure data factory window so that visual impaired user could easily such list items.

![MAS1 4 3_Azure data factory](https://user-images.githubusercontent.com/72871089/101166960-28d87700-365f-11eb-95ea-35e3ba3184f4.png)

**Comment by craxal** [comment]
The inaccessible color comes from the OS. Although it's possible to style select options, the hover state of an option is not something we have any direct control over. I'm going to accessibility office hours to see if there's a reasonable solution we can use.

**Comment by craxal** [comment]
Accessibility Office Hours confirmed that `<select>` element styling is controlled by the underlying framework, which is Chromium. An issue has or is being filed against Chromium, but on the Storage Explorer side, there's nothing we can do.

The ADF extension does not appear to be using native `<select>` elements. It looks like they were trying to mimic the colors from Storage Explorer's `<select>` elements, so they should be able to make the necessary adjustments.

**Comment by craxal** [comment]
As discussed in Accessibility Office Hours, as far as Storage Explorer is concerned, this bug is invalid because the root cause is in Chromium.
